src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -2088,8 +2048,9 @@ protected override void VisitArguments(ImmutableArray<BoundExpression> arguments
             ImmutableArray<int> argsToParamsOpt,
             bool expanded)
         {
-            // PROTOTYPE(NullableReferenceTypes): What about conversions here?
-            VisitArguments(node, arguments, refKindsOpt, method is null ? default : method.Parameters, argsToParamsOpt, expanded);
+            ImmutableArray<Conversion> conversions;
+            (arguments, conversions) = RemoveArgumentConversions(arguments, refKindsOpt);
```

Comment:
Not related to this PR: Just to confirm, we don't yet handle user-defined conversions with nullability involved, right?